### DISCUSSION TITLE
Add zoom in to header with no content

### DISCUSSION THREAD

**Discussion by charleshan** [vslinko/obsidian-zoom#61] [Ideas]
It looks like we can do this by creating a bullet, zooming in, and clicking on the header, but there's no way to do this from the top level. The shortcut to zoom in doesn't work.

e.g.
```
# Header 1
## Header 2
- Bullet point. You can zoom in here and change the zoom level to one of the headers but you can do this without this bullet point.
```

**Comment by vslinko** [comment]
@charleshan thank you for the feedback!

I can't reproduce the issue. Could you please check that the "Fold heading" setting is enabled?

<img width="1222" alt="Screenshot 2022-04-06 at 09 27 11" src="https://user-images.githubusercontent.com/205823/161909443-bb7b2c80-bd11-4f13-a409-3edf9bfb7fbf.png">

**Comment by charleshan** [comment]
"Fold heading" is enabled.

I was wrong about my initial observation. It's not that we need a bullet point but any content under the header to zoom in. Can you try to zoom into a header without having anything below it?

**Comment by vslinko** [comment]
Oh, yes.

You can't zoom in if there's no content under the header. That's because my mechanism relies on the built-in folding functionality and repeats its behavior.

I can do a hack and support this use case, but in priority of feature requests.
